itests/src/test/resources/testconfiguration.properties

```diff
@@ -1251,4 +1251,11 @@ erasurecoding.only.query.files=\
 # tests that requires external database connection
 externalDB.llap.query.files=\
   dataconnector.q,\
-  dataconnector_mysql.q
\ No newline at end of file
+  dataconnector_mysql.q
+
+iceberg.llap.query.files=\
```

Comment:
We might want to use specific directories specified by the driver and then we do not have to use `testconfiguration.properties`.

Reply:
The reason I introduced this like that was to execute vectorized_iceberg_read.q on both LLAP and non-LLAP configs, without copying it over into another directory. Also, we only have very few iceberg q files yet, and just one other LLAP related q file.